docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-product-options-feature-walkthrough.md

```diff
@@ -13,9 +12,26 @@ To learn more about the feature and to find out how end users use it, see [Marke
 
 {% endinfo_block %}
 
-## Related Developer articles
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/d8882366-b2dd-4d6c-b401-01db47a00481.png?utm_medium=live&utm_source=custom)
+
+| NAME | DESCRIPTION | 
+| --- | --- |
+| [MerchantProductOption](https://github.com/spryker/merchant-product-option) | Provides merchant product option main business logic and persistence. |
+| [MerchantProductOptionDataImport](https://github.com/spryker/merchant-product-option-data-import) | Provides data import functionality for merchant product options. |
+| [MerchantProductOptionStorage](https://github.com/spryker/merchant-product-option-storage) | Provides publish and sync functionality for merchant product options. |
+| [MerchantProductOptionGui](https://github.com/spryker/merchant-product-option-gui) | Provides backoffice UI for merchant product options management. |
+| [ProductOption](https://github.com/spryker/product-option) | Provides additional layer of optional items that can be sold with the actual product. |
+| [ProductOptionStorage](https://github.com/spryker/product-option-storage) | Provides publish and sync functionality for product options. |
+| [Shop.ProductOptionWidget](https://github.com/spryker-shop/product-option-widget) | Provides widgets for displaying product options. |
 
+## Domain model
+
+![Domain Model](https://confluence-connect.gliffy.net/embed/image/90a0e5bc-a0d9-4cb2-a215-c5d08a786115.png?utm_medium=live&utm_source=custom)
```

Comment:
1. Since both objects are part of the feature, please change the color to blue.
2. Please add the `ProductOption` as part of the diagram.
3. Diagram relation is wrong, ProductOptionGroup can not have several MerchantProductOptionGroup's.

Reply:
Redrawn the domain model diagram according the requirements